en/hosting_guide/site_creation.rst

```diff
@@ -0,0 +1,106 @@
+Website creation
+================
+
+Creating a new site with Open Orchestra requires three steps. Additionnally, you can add some more
+steps to enhance the site features. Those steps are the following:
+
+* Declaring the website in Open Orchestra
+* Dealing with rights
+* Contributing the home page
+* Creating an Apache virtual host
```

Comment:
I wouldn't say "apache" as you don't want to limit yourself. This section can contain an apache example but should be server agnostic